### PR TITLE
Use DI for IAppManager to encryption migration

### PR DESCRIPTION
**Pull request by MorrisJobke**
One step towards removing `\OC_App::isEnabled`

### CODE REVIEW

File: tests/Settings/Controller/EncryptionControllerTest.php [hunk 1]
```diff
@@ -75,6 +78,8 @@ public function setUp() {
 			->disableOriginalConstructor()->getMock();
 		$this->logger = $this->getMockBuilder('\\OCP\\ILogger')
 			->disableOriginalConstructor()->getMock();
+		$this->appManager = $this->getMockBuilder('\\OCP\\App\\IAppManager')
```
Comment (ChristophWurst) [inline tests/Settings/Controller/EncryptionControllerTest.php]: I guess you're using this old-fashioned coding style for consistency reasons? At some point we should consistently migrate this to the `::class` property to allow better tooling/IDE support.

**Review by ChristophWurst — APPROVED** [review]
Way to go! Keep those cleanup coming, I like them 🚀

**Comment by MorrisJobke** [comment]
I need to look into the tests here as well.

**Comment by MorrisJobke** [comment]
Tests fixed and run fine locally.

**Review by schiessle — APPROVED** [review]

**Pull request merged**